packages/api/endpoints/rules.js

```diff
@@ -62,6 +65,9 @@ async function post(req, res) {
   } catch (e) {
     if (e instanceof RecordDoesNotExist) {
       const r = await model.create(data);
+      req.rulesRecord = r;
```

Comment:
Can we put these inside the `inTestMode()` block?

Reply:
changed